plugins/modules/vmware_vc_infraprofile_info.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Ansible Project
+# Copyright: (c) 2019, Naveenkumar G P <ngp@vmware.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = r'''
+---
+module: vmware_vc_infraprofile_info
+short_description: List, Export, Validate and import VMware Vcenter infra profile configs.
+description:
+- Module to manage VMware Vcenter infra profile configs.
+- Vcenter infra profile Library feature is introduced in vSphere 7.0 version, so this module is not supported in the earlier versions of vSphere.
+- All variables and VMware object names are case sensitive.
+version_added: '2.9'
+author:
+- Naveenkumar G P (@ngp)
+notes:
+- Tested on vSphere 6.5, 6.7
```

Comment:
Sorry for catching this in the earlier review round, in the description, it says this feature is available in vSphere 7. I think this line should be changed to reflect the same. Also, add a check in the code for vCenter version.